test/e2e/multiparty/tokens.go

```diff
@@ -370,3 +370,191 @@ func (suite *TokensTestSuite) TestE2ENonFungibleTokensSync() {
 	accountPools = suite.testState.client2.GetTokenAccountPools(suite.T(), suite.testState.org2key.Value)
 	assert.Equal(suite.T(), *poolID, *accountPools[0].Pool)
 }
+
+func (suite *TokensTestSuite) TestE2EFungibleTokensWithRemoteNameAsync() {
+	defer suite.testState.done()
+	config1 := e2e.ReadConfig(suite.T(), suite.testState.configFile1)
+	e2e.AddPluginRemoteName(config1, "tokens", "testremote")
+	e2e.WriteConfig(suite.T(), suite.testState.configFile1, config1)
+
+	config2 := e2e.ReadConfig(suite.T(), suite.testState.configFile2)
+	e2e.AddPluginRemoteName(config2, "tokens", "testremote")
+	e2e.WriteConfig(suite.T(), suite.testState.configFile2, config2)
```

Comment:
We probably want the local plugin names to be different on the two nodes (otherwise it might be ignoring the remote name and still just happen to work because the local names match).

This test also might need to be in a suite of its own, since it makes pretty invasive changes to the config.